presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -1,10 +1,11 @@
 =============
-Web Interface
+Presto Web UI
 =============
 
-Presto provides a web interface for monitoring and managing queries.
-The web interface is accessible on the Presto coordinator via HTTP/HTTPS,
-using the corresponding port number specified in the coordinator :ref:`config_properties`.
+Presto provides a web-based user interface for monitoring a Presto cluster and
+managing queries, the Presto Web UI. The web UI is accessible on the Presto
```

Comment:
Let's remove the second part and add (UI) after "user interface"

> Presto provides a web-based user interface (UI) for monitoring a Presto cluster and managing queries.

Reply:
Done